cmd/promtool/main.go

```diff
@@ -747,3 +766,38 @@ func (j *jsonPrinter) printLabelValues(v model.LabelValues) {
 	//nolint:errcheck
 	json.NewEncoder(os.Stdout).Encode(v)
 }
+
+// BackfillRule backfills rules from the files provided
```

Comment:
full stop

Reply:
what do you mean by "full stop"?

Reply:
The comments must ends with a full stop (a dot): .

Reply:
ah thanks for the explanation, will fix